core/src/main/java/io/fabric8/maven/core/access/ClusterAccess.java

```diff
@@ -72,6 +73,18 @@ public String getNamespace() {
         return namespace;
     }
 
+    /**
+     * Returns true if this cluster is a traditional OpenShift cluster with the <code>/oapi</code> REST API
+     * or supports the new <code>/apis/image.openshift.io</code> API Group
+     */
```

Comment:
understand. But when does it happen that isOpenShift() == true but not supporting neither the new nor the old OpenShift imagestream API ? (or in other words: when is isOpenShift() alone not enough)